### ISSUE TITLE
Squirrelmail with Greenmail ERROR “Bad or malformed request”

### ISSUE THREAD

**Issue by marcelmay** [greenmail-mail-test/greenmail#2]
http://stackoverflow.com/questions/21471726/squirrelmail-with-greenmail-error-bad-or-malformed-request

**Comment by marcelmay** [comment]
With latest commit for #103, Squirrelmail at least shows messages. There is still an issue when fetching message details:

```
ERROR: Bad or malformed request.
Query: FETCH 1 BODY[HEADER]
Server responded: Expected:'(' found:'B' Command should be '<tag> UID <fetch-command>|<store-command>|<copy-command>|<search-command>'
```

**Comment by marcelmay** [comment]
Just committed another fix #104 affecting Squirrelmail.

- Composing mail works
- Listing mails in inbox works
- Opening mail works
- Search Body/Subject/Everywhere works
- Search From/CC/TO ***fails*** :
```
FETCH (FLAGS UID RFC822.SIZE INTERNALDATE BODY.PEEK[HEADER.FIELDS (Date To Cc From Subject X-Priority Importance Priority Content-Type)])
Server responded: Invalid character: '(' Command should be '<tag> UID <fetch-command>|<store-command>|<copy-command>|<search-command>'
```

**Comment by marcelmay** [comment]
Works with 1.6.0-SNAPSHOT / 1.5.4
